package.json

```diff
@@ -16,6 +16,7 @@
     "webdriver": "webdriver-manager update && webdriver-manager start"
   },
   "devDependencies": {
+    "@kennsippell/medic-conf": "^3.0.7",
```

Comment:
TODO reminder.

Reply:
medic-conf package is published! Reverted this